consensus/src/round_manager.rs

```diff
@@ -145,6 +146,7 @@ pub enum VerifiedEvent {
     // network messages
     ProposalMsg(Box<ProposalMsg>),
     VerifiedProposalMsg(Box<Block>),
+    VerifiedDelayedQcMsg(Box<DelayedQcMsg>),
```

Comment:
I don't think we should add it to VerifiedEvent, it probably can be in its own channel with its own (simpler) type, it's confusing to see this under // network messages